packages/rolldown/tests/fixtures/sourcemap/inner/_config.ts

```diff
@@ -25,7 +25,7 @@ export default defineTest({
     const map = JSON.parse(output.output[1].source)
     expect(map.file).toMatchInlineSnapshot(`"main.js"`)
     expect(map.mappings).toMatchInlineSnapshot(
-      `";;AAAO,MAAM,MAAM;;;;ACEnB,QAAQ,IAAI,IAAI"`,
+      `";;AAAA,MAAa,MAAM;;;;ACEnB,QAAQ,IAAI,IAAI"`,
```

Comment:
I have verified the sourcemap is correct.
![image](https://github.com/rolldown/rolldown/assets/17974631/2ba72e87-f9ff-49c8-9f89-545194f8e382)